### PR TITLE
new logo

### PR DESCRIPTION
**Pull request by efriis**

### CODE REVIEW

**Comment by hinthornw** [comment]
image
Hm I can't see the logo in light mode

File: docusaurus.config.js [hunk 1]
```diff
@@ -9,13 +9,13 @@ require("dotenv").config();
...
   tagline: "LangSmith",
-  favicon: "img/favicon.ico",
+  favicon: "img/favicon.png",
   customFields: {
     supabaseUrl: process.env.NEXT_PUBLIC_SUPABASE_URL,
     supabasePublicKey: process.env.NEXT_PUBLIC_SUPABASE_PUBLIC_KEY,
   },
   // Set the production url of your site here
-  url: "https://smith.langchain.com/", // TODO: also make configurable with dev deployment
+  url: "https://docs.smith.langchain.com/", // TODO: also make configurable with dev deployment
```
Comment (efriis) [inline docusaurus.config.js:18]: this was wrong

File: docusaurus.config.js [hunk 2]
```diff
@@ -9,13 +9,13 @@ require("dotenv").config();
 const config = {
   title: "🦜️🛠️ LangSmith",
   tagline: "LangSmith",
-  favicon: "img/favicon.ico",
+  favicon: "img/favicon.png",
```
Comment (efriis) [inline docusaurus.config.js:12]: pngs work for favicons now

File: package.json [hunk 3]
```diff
@@ -24,9 +24,9 @@
     "gen:supabase": "npx supabase gen types typescript --project-id 'xsqpnijvmbodcxyapnyq' --schema public > ./src/supabase.d.ts"
   },
   "dependencies": {
-    "@docusaurus/core": "2.4.0",
-    "@docusaurus/preset-classic": "2.4.0",
-    "@docusaurus/remark-plugin-npm2yarn": "^2.4.0",
+    "@docusaurus/core": "2.4.3",
```
Comment (efriis) [inline package.json:27]: was doing this to debug what ended up being base url issue above, and good to do anyways

File: docusaurus.config.js [hunk 4]
```diff
@@ -91,9 +91,9 @@ const config = {
         theme: require("prism-react-renderer/themes/vsLight"),
         darkTheme: require("prism-react-renderer/themes/vsDark"),
       },
-      image: "img/langsmith-logo.svg",
+      image: "img/langsmith-preview.png",
```
Comment (efriis) [inline docusaurus.config.js:94]: this will render as the "link preview"

**Review by hinthornw — APPROVED** [review]

**Pull request merged**